tests/quantization/test_precision_init.py

```diff
@@ -711,7 +711,8 @@ def get_quantization_config_with_ignored_scope():
 @pytest.mark.parametrize(('config_creator', 'ref_values'), (
     [
         get_quantization_config_without_range_init,
-        (1.75, pytest.approx(1.07, abs=1e-2), (1, 2), (1, 4), (1, 1.12))
+        (pytest.approx(1.57, abs=1e-2), pytest.approx(1.15, abs=1e-2), (1, 2), (1, 4),
```

Comment:
@ljaljushkin I deleted BatchNorms from FLOPs calculating and fix test in quantization used FLOPs count. Please, check that it is correct now.

Reply:
It's strange that FLOPS were changed after deleting BN because the test model doesn't have it.
Looks like there's a change in the formula for Conv and Linear

Reply:
Could you verify that's discrepancy happens because of considering `bias`?
```
if module.bias is not None:
            flops_count += np.prod(output.shape[1:])
```

Reply:
@ljaljushkin For Conv and Linear bias addition was added to FLOPs calculation. Check it in `nncf/utils.py`

Reply:
Ok, I will re-check it.

Reply:
Biased of Conv and FC should be also excluded from FLOPS

Reply:
OK, biases are deleted from FLOPs calculation and tests for quantization now work without fixes.